api/token.go

```diff
@@ -27,8 +27,8 @@ type GoTrueClaims struct {
 	AppMetaData                   map[string]interface{} `json:"app_metadata"`
 	UserMetaData                  map[string]interface{} `json:"user_metadata"`
 	Role                          string                 `json:"role"`
-	AuthenticatorAssuranceLevel   string                 `json:"aal"`
-	AuthenticationMethodReference []models.AMREntry      `json:"amr"`
+	AuthenticatorAssuranceLevel   string                 `json:"aal,omitempty"`
+	AuthenticationMethodReference []models.AMREntry      `json:"amr,omitempty"`
```

Comment:
if the feature flag is on (meaning MFA is enabled), these claims will never be empty right?

Reply:
Yup, you're right if  the feature flag is on the claims won't be empty. If it's off we don't want to show these fields though